Tasks/PublishPipelineMetadataV0/package-lock.json

```diff
@@ -1,8 +1,721 @@
 {
   "name": "publishpipelinemetadatav0",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
https://github.com/microsoft/azure-pipelines-tasks/pull/17277#discussion_r1032273934
I guess the same here.

Reply:
Fixed